### DISCUSSION TITLE
TP-Link (TL-WN832n) not working on macOS Monterey 12.0.1

### DISCUSSION THREAD

**Discussion by jbagaresgaray** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#146] [General]
Just freshly installed my macOS Monterey 12.0.1 on my AMD Ryzen 7 3700x Desktop but the Wireless-USB-OC-Big-Sur-Adapter V11 package didn't recognize my TP-Link TL WN823N

I keep on installing the package but I could not find the kext is installed on the open core.

By the way this is box of my wireless usb adapter to be specific 

![image](https://user-images.githubusercontent.com/3625802/140258181-3f1cfea4-9505-482e-9847-2e9232a62602.png)

**Comment by chris1111** [comment]
Hi Philip Issue is not a discussion ; [see are Rules](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/discussions/19)
This discussion will be deleted but by the way I have this adapter for long time and its work, Your only have to use it on USB 2.0 port, try to change the port thats I have to do for its work just now.

![Screen Shot 2021-11-04 at 7 36 42 AM](https://user-images.githubusercontent.com/6248794/140307549-fa8dc449-fd1d-405f-8919-e1433cb22ed6.png)
